samples/order-management-system/README.md

```diff
@@ -0,0 +1,156 @@
+# Order Management Scenario
+
+## Scenario
+
+This scenario is about a order management system. It consists of two parties. Those are a customer and an order management system(OMS).
+In our sample we are hosting our OMS in  WSO2 Application Server. Customer would interact with OMS to place orders, view orders, make payments and view payments.
```

Comment:
WSO2 Application Server is out of the stack now?

Reply:
Yes, For the moment we will be using this until we will be able to develop a SOAP backend from ballerina.